Repository/Vcs/GitDriver.php

```diff
@@ -35,4 +36,24 @@ public function getComposerInformation($identifier)
 
         return ProcessUtil::getComposerInformation($this->cache, $this->infoCache, $this->repoConfig['asset-type'], $this->process, $identifier, $resource, sprintf('git show %s', $resource), sprintf('git log -1 --format=%%at %s', escapeshellarg($identifier)), $this->repoDir, '@');
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function initialize()
+    {
+        if (isset($this->config->get('fxp-asset')['git-driver']['skip-update'])) {
```

Comment:
When I dump (print_r) `repoConfig` in `initialize()` I get:

```
Array
(
    [url] => https://github.com/twbs/bootstrap.git
    [asset-type] => bower
    [filename] => bower.json
)
```

There's no `asset-repository-manager` key?!